### PR TITLE
feat: switch dashboard pages

### PR DESCRIPTION
**Pull request by SiongSng**
# Description
Switch pages on the dashboard.
I create a new Vue component that named `PageDashboard` to manage pages (overview/class/lesson/note/exam), and test the effect of switch pages (click the button on the navigation bar)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/mobile/dashboard/OverviewContent.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import RecentCard from '@/components/Mobile/Overview/RecentCard.vue';
+import RecentCard from '@/components/mobile/dashboard/RecentCard.vue';
 import { useUserStore } from '@/stores/models/user';
 
 const user = useUserStore().info;
 </script>
 
 <template>
-  <div class="mx-[30px] text-center">
+  <div class="max-h-[calc(100svh-149px)] overflow-y-auto px-[6%] text-center">
```
Comment (SiongSng) [inline src/components/mobile/dashboard/OverviewContent.vue:10]: 由於先前留下的技術債，暫時以減去實際的大小來處理。

File: src/pages/dashboard/PageClass.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup></script>
+
+<template>
+  <div
+    class="absolute right-0 top-[77px] h-[calc(100svh-77px)] w-[calc(100svw-104px)] phone:hidden"
```
Comment (SiongSng) [inline src/pages/dashboard/PageClass.vue:6]: 由於先前留下的技術債，暫時以減去實際的大小來處理。

File: src/pages/dashboard/PageExam.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup></script>
+
+<template>
+  <div
+    class="absolute right-0 top-[77px] h-[calc(100svh-77px)] w-[calc(100svw-104px)] phone:hidden"
```
Comment (SiongSng) [inline src/pages/dashboard/PageExam.vue:6]: 由於先前留下的技術債，暫時以減去實際的大小來處理。

File: src/pages/dashboard/PageLesson.vue [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup></script>
+
+<template>
+  <div
+    class="absolute right-0 top-[77px] h-[calc(100svh-77px)] w-[calc(100svw-104px)] phone:hidden"
```
Comment (SiongSng) [inline src/pages/dashboard/PageLesson.vue:6]: 由於先前留下的技術債，暫時以減去實際的大小來處理。

File: src/pages/dashboard/PageNote.vue [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<script lang="ts" setup></script>
+
+<template>
+  <div
+    class="absolute right-0 top-[77px] h-[calc(100svh-77px)] w-[calc(100svw-104px)] phone:hidden"
```
Comment (SiongSng) [inline src/pages/dashboard/PageNote.vue:6]: 由於先前留下的技術債，暫時以減去實際的大小來處理。

File: src/scss/global.scss [hunk 6]
```diff
@@ -20,6 +20,10 @@ html,
 #app {
   width: 100%;
   height: 100%;
+
+  .dark & {
+    background-color: $DarkBlack; // For Webkit browsers.
```
Comment (SiongSng) [inline src/scss/global.scss]: 用於修正在 Safari 中滑動頁面到超越外圍時會有一個動畫出現，其中的顏色與前端本身的背景顏色不搭。

**Review by HACO8888 — APPROVED** [review]

**Review by banahaker — APPROVED** [review]

**Pull request merged**